<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>주정차단속관리</string>
  </property>
  <property name="locale">
   <locale language="Korean" country="SouthKorea"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>540</width>
      <height>335</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>540</width>
      <height>345</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>615</y>
      <width>540</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>615</y>
      <width>540</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BareunDotum</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>차량번호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>400</width>
      <height>250</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>자료등록</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>20</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string> 사진등록</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>편성정보</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>자료검색</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>단속위치</string>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>차량번호</string>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>16</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>999999</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>등록번호</string>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>단속일시</string>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>16</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>293</x>
        <y>130</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>편성정보</string>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>71</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>자료검색</string>
      </property>
      <property name="shortcut">
       <string>F3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>자료편집</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>[F4] 자료저장</string>
      </property>
      <property name="shortcut">
       <string>F4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>[F5] 자료삭제</string>
      </property>
      <property name="shortcut">
       <string>F5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>181</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>181</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>181</width>
        <height>33</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd hh:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>차량번호</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>95</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>단속위치</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>135</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>단속일시</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>175</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>편성정보</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>55</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>사진1변경</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>95</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>사진2변경</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_12">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>135</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>사진3변경</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>175</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>사진4변경</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>자료확정</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>[F6] 단속확정</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>[F7] 엑셀변환</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>98</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>환경설정</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>사진등록폴더</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>자료작업폴더</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>백업파일폴더</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>자료확정폴더</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>98</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>도움말</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>98</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>프로그램정보</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>30</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>30</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>30</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>30</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>751</width>
      <height>251</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>dateTimeEdit_3</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>checkBox_12</tabstop>
  <tabstop>checkBox_13</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_12</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton_15</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton_16</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>pushButton_17</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>pushButton_18</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_13</tabstop>
  <tabstop>pushButton_14</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_10</tabstop>
  <tabstop>pushButton_11</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
